packages/server/utils/stripe/StripeManager.ts

```diff
@@ -50,6 +50,16 @@ export default class StripeManager {
     })
   }
 
+  async createPaymentIntent(amount: number) {
+    return this.stripe.paymentIntents.create({
+      amount,
+      currency: 'usd',
+      automatic_payment_methods: {
+        enabled: true
+      }
```

Comment:
+1 this will have to be revisited to make sure we're handling all the necessary webhooks for selected payment methods